drizzlepac/hlautils/get_git_rev_info.py

```diff
@@ -92,6 +92,10 @@ def get_rev_id(local_repo_path):
     full git revision ID of the specified repository if everything ran OK, and "FAILURE" if something went
     wrong.
     """
+
+    # Ensure the return variable is initialized
+    rv = "FAILURE: git revision info not found"
```

Comment:
This specific change looks to be just what was needed.  However, it duplicates line 111 which could just be replaced with a 'pass' statement instead.  Also, the lone 'raise' in line 109(?) is not strictly proper Python based on Section 8.4 of the Python documentation.  Instead, it should be something like 'raise ValueError("Git version not found")'.  This code will work, but it does not strictly follow good Python practice as it is now.

Reply:
Addressed issues.